remotespark/datawidgets/encoding.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2015  aggftw@gmail.com
+# Distributed under the terms of the Modified BSD License.
+
+
+class Encoding(object):
```

Comment:
Given that the kernels don't have anything to do with Altair anymore, I don't see why that's necessary.